### PR TITLE
T177934 Resolve many content pages issues

### PR DESCRIPTION
**Pull request by wiese**
For T179930, T179929, T179936, T179931

### CODE REVIEW

File: skins/cat17/templates/Display_Page_Layout.twig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% extends 'Base_Layout.html.twig' %}
 
-{% block page_identifier %}{$ page_id $}{% endblock %}
+{% block page_identifier %}page-{$ page_id $}{% endblock %}
```
Comment (wiese) [inline skins/cat17/templates/Display_Page_Layout.twig]: I don't particularly like the prefix but without it in the CSS selectors I am afraid to involuntarily match styles. Will follow-up and consolidate once all pages work the same way (prepend it to only the identifier rendered onto the body, not the page_identifier variable).

File: skins/cat17/src/sass/layouts/pages/privacy_protection.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.page-privacy_protection {
+	.content {
+		li {
+			list-style-position: inside;
```
Comment (gbirke) [inline skins/cat17/src/sass/layouts/pages/privacy_protection.scss]: The list at the top of the page looks ok, but when you have longer lines, they are not indented correctly when you have a line break in the list (see section "Betroffene Personengruppen und Daten oder Datenkategorien")

Comment (wiese) [reply skins/cat17/src/sass/layouts/pages/privacy_protection.scss]: Looks pretty much like in 10h16 on prod - please define "correctly" (T179936).

Comment (gbirke) [reply skins/cat17/src/sass/layouts/pages/privacy_protection.scss]: padding-left: 1.5em;
    text-indent: -1.5em;

looks nice

**Comment by wiese** [comment]
Rebased and amended (T179936) follwing @gbirke's comment.

**Pull request merged**